sdk/storagesync/Azure.ResourceManager.StorageSync/src/autorest.md

```diff
@@ -8,8 +8,8 @@ azure-arm: true
 csharp: true
 library-name: StorageSync
 namespace: Azure.ResourceManager.StorageSync
-require: https://github.com/Azure/azure-rest-api-specs/blob/2614c5038cb384e864c1608e76be61a5ee84cb02/specification/storagesync/resource-manager/readme.md
-tag: package-2020-09-01
+require: https://github.com/Azure/azure-rest-api-specs/blob/d1eee5499dbf9281debdc90c4f4cbc7470fb8d6d/specification/storagesync/resource-manager/readme.md
```

Comment:
Please update the CHANGELOG.md and give the date of when do you want to release the new package

Reply:
Sounds good, just updated the CHANGELOG.md. 

Could we get this released at the next earliest date? I slotted the release for 2022-10-21 for now but it can be changed depending on your availability for publishing the release. Thanks!